package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "destructors-ec-bn",
   "version": "1.0.0",
-  "type": "module",
   "description": "Destructors ecommerce team project",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js",
-    "dev": "nodemon src/index.js"
+    
```

Comment:
You had removed start script for production

Reply:
also you can save time by creating script command for migrating database 
"migrate": "sequelize db:migrate"

Reply:
thank you @leandreAlly,